packages/storage/amplify_storage_s3/example/integration_test/utils/tear_down.dart

```diff
@@ -36,6 +36,27 @@ void addTearDownPaths(List<StoragePath> paths) {
   );
 }
 
+/// Adds a tear down to remove the same object in multiple [buckets].
+void addTearDownMultiBucket(StoragePath path, List<StorageBucket> buckets) {
```

Comment:
nit: I don't see this used anywhere. Is a different PR using this code?

Reply:
Yup, `uploadFile` integ tests use it.